Makefile

```diff
@@ -85,18 +86,32 @@ $(LUCENE_ANALYZER_JAR):
 
 jars: $(INDEXER_JAR) $(TAGSOUP_JAR) $(LUCENE_ANALYZER_JAR) $(LUCENE_CORE_JAR)
 
-docs/index.html: $(SOURCES)
-	mkdir -p docs
+webhelp/index.html: $(SOURCES)
```

Comment:
Why change this directory name? It will be very inconvient, as we'll have to change all the jobs in the CI to copy a different directory, etc... Could we not just keep the "main" name as is?